docs/src/pages/styles/basics/basics.md

```diff
@@ -1,8 +1,13 @@
 # @material-ui/styles
 
-<p class="description">You can use Material-UI's styling solution in your app, whether or not you are using Material-UI components.</p>
+<p class="description">The legacy styling solution of Material-UI.</p>
 
-> **Note**: `@material-ui/styles` is the _**legacy**_ styling solution for Material-UI. It is deprecated in v5. It depends on [JSS](https://cssinjs.org/) as a styling solution, which is not used in the `@material-ui/core` anymore. If you don't want to have both emotion & JSS in your bundle, please refer to the [`@material-ui/system`](/system/basics/) documentation which is the recommended alternative.
+> ⚠️ `@material-ui/styles` is the legacy styling solution for Material-UI.
+> It's **deprecated** in v5, which is not used by the other packages anymore.
+> It depends on [JSS](https://cssinjs.org/).
+> If you don't want to have both emotion & JSS in your bundle, please refer to the [`@material-ui/system`](/system/basics/) documentation which is the recommended alternative.
```

Comment:
```suggestion
> If you don't want to have both Emotion & JSS in your bundle, please refer to the [`@material-ui/system`](/system/basics/) documentation which is the recommended alternative.
```

Reply:
@Andarist how would you wish https://github.com/emotion-js/emotion to be referenced in the documentation of Material-UI?

We have used "emotion" so far, but maybe we shouldn't have. For instance, https://emotion.sh/docs/introduction

<img width="403" alt="Capture d’écran 2021-08-09 à 23 58 00" src="https://user-images.githubusercontent.com/3165635/128779758-6f060893-8d97-44b0-9667-6d4d2f1a3760.png">

https://emotion.sh/docs/css-prop

<img width="391" alt="Capture d’écran 2021-08-09 à 23 59 14" src="https://user-images.githubusercontent.com/3165635/128779869-df122297-922a-4389-8702-a290adbdd0b8.png">

Reply:
I think Emotion is fine - it makes it more obvious that it's the name of something (like a library). I probably should unify this across the docs.